workflow/applications/gfs_cycled.py

```diff
@@ -254,8 +256,9 @@ def get_task_names(self):
             gfs_tasks += ['gempak']
             gfs_tasks += ['gempakmeta']
             gfs_tasks += ['gempakncdcupapgif']
-            gfs_tasks += ['npoess_pgrb2_0p5deg']
-            gfs_tasks += ['gempakpgrb2spec']
+            if self.do_goes:
+                gfs_tasks += ['npoess_pgrb2_0p5deg']
+                gfs_tasks += ['gempakpgrb2spec']
```

Comment:
Does the task name `gempakpgrb2spec` need to be consistent with the one in rocoto/gfs_tasks.py? If so, it should be `gempakgrb2spec` without "p".

Reply:
All that matter is that is matches the function name in `gfs_tasks.py` (which is does). That function specifies the bash script to use as the job entrypoint.